test/src/e2e.dynval/setup.ts

```diff
@@ -236,6 +236,39 @@ export async function createNodes(options: {
     }
 }
 
+export async function selfNominate(
+    sdk: SDK,
+    validator: ValidatorConfig["signer"],
+    deposit: number
+): Promise<H256> {
+    const tx = stake.createSelfNominateTransaction(sdk, deposit, "").sign({
+        secret: validator.privateKey,
+        seq: await sdk.rpc.chain.getSeq(validator.platformAddress),
+        fee: 10
+    });
+
+    return await sdk.rpc.chain.sendSignedTransaction(tx);
```

Comment:
How about calling `waitForTx` too?

Reply:
To do it, we need to receive a value of type `CodeChain` that is a very heavy object in arguments.
Is it too complicated object to pass to this function?

Reply:
`changeParams` already receives the `CodeChain` instance.
It's too big component though. Let's leave it as is for now.